server/controller/user.js

```diff
@@ -8,23 +8,23 @@ dotenv.config();
 const secret = process.env.SECRET;
 class UserController{
     static signUp (req, res, next) {
-        const query = 'INSERT INTO users(firstname, lastname, about, email, password, confirmpassword, created_at) VALUES($1, $2, $3, $4, $5, $6, Now() ) RETURNING *';
+        var email = req.body.email.split(' ').join('')
+        const query = 'INSERT INTO users(firstname, lastname, about, email, password, created_at) VALUES($1, $2, $3, $4, $5, Now() ) RETURNING *';
         const values =[
             req.body.firstname, 
             req.body.lastname, 
             req.body.about, 
-            req.body.email, 
+            email, 
             req.body.password,
-            req.body.confirmPassword
         ]
         client.query(query, values, (err, data) => {
-            if (err) return res.status(409).send({
+            if (err) return res.status(500).send({
                     success: false,
-                    message: "User already exist."
+                    message: "User Already exist."
```

Comment:
If there is any error now, the error message the user would be getting is user already exist, irrespective of what the error is. I felt like the 500 is server error, if you want to validate of user already exist then you should do a check to do that, and the status code for that would now be 409.